### ISSUE TITLE
[REQUEST] Job flag to not store hit types

### ISSUE THREAD

**Issue by JReming85** [openbullet/OpenBullet2#410]
It might just be me but I feel like I am going in and clearing out hundreds of TOChecks and NONEs from the Hits DB all the time and think it might be beneficial to have a flag or option box to not log certain hits like TOCheck or any custom ones added.

**Comment by openbullet** [comment]
Honestly if you get that many ToChecks maybe the config needs to be fixed ^^ There is already an option to only store hits (job options -> hit outputs) but I don't remember if I also added it for the db. I will check.

**Comment by JReming85** [comment]
To be fair its mostly when I am building and refining my configs, mostly for cold testing sites where I have no info before hand.

But I appreciate it.

**Comment by openbullet** [comment]
Yes on some hit outputs there is a checkbox to save only hits. Also on the file system hit output, OB2 creates different files for different statuses so hits will be in a separate file, I think for the time being you could use that until I implement this issue.

**Comment by JReming85** [comment]
Awesome thanks for the help and please dont feel like this is something that needs to be done right away.
